storage/innobase/handler/ha_innodb.cc

```diff
@@ -346,7 +346,7 @@ TYPELIB innodb_linux_aio_typelib = {
 	array_elements(innodb_linux_aio_names) - 1,
 	"innodb_linux_aio_typelib",
 	innodb_linux_aio_names,
-	NULL
+	NULL, NULL
 };
 #endif
```

Comment:
What is this change about? Why is it needed? There is no mention of this in the commit message.